app/src/androidTest/java/com/appunite/loudius/LoginScreenTest.kt

```diff
@@ -46,12 +55,24 @@ class LoginScreenTest {
 
     @Test
     fun whenTheLoginScreenIsVisibleThenTheLogInButtonIsVisible() {
+        Intents.init()
```

Comment:
Suggestion/Question: I'm wondering if `@Test` is the best place for `Intents.init()`. Maybe we should put it in `@Before` function in case we need it in another test.

Reply:
If I needed this in a different test I would probably do it, but currently, I don't have any other test. Also, I am not sure if it affects tests in any way if we don't use `intended/intending`, probably no, but I don't want to mess up new tests with it unintentionally

Reply:
I suggest using `IntentsRule`:
```
@get:Rule(order = 1)
val intentsRule = IntentsRule()
```

Especially because in case of a test failure, `Intents.release()` is going to be released. 

It shouldn't be a problem if intents is not used in a given test. It should'n make any unnecessary action.